### DISCUSSION TITLE
How to change the theme of the context menu?

### DISCUSSION THREAD

**Discussion by 0-BlackSpectrum-0** [moudey/Shell#174] [Q&A]
As shown in the images:
Gradient:
https://github.com/moudey/Shell/blob/main/screenshots/gradient.png
Acrylic:
https://github.com/moudey/Shell/blob/main/screenshots/acrylic.png

how do I apply these themes?

**Comment by moudey** [comment]
Add this codes to `theme` section in `shell.nss` file

Gradient
```
background
{
    opacity=0
    gradient
    {
        enabled = 1
        linear = [0, 100, 0, 0]
        stop = [
            [0, color.accent_dark3, 90],
            [.5, color.accent, 90],
            [1, color.accent_dark3, 90]
        ]
    }
}
```
Acrylic
```
background
{
    opacity=0
    effect=[3, color.accent, 50]
}
```

Save changes, press `CTRL` + `RIGHT-CLICK` or restart Explorer to reload settings.

https://nilesoft.org/docs/syntax/set

**Reply by bachig26** [reply]
I tried these same for acrylic, but I can't able to reproduce the theme in win10. do I need to include the gradient too for acrylic theme?

**Reply by moudey** [reply]
>I tried these same for acrylic, but I can't able to reproduce the theme in win10. do I need to include the gradient too for acrylic theme?

Use the blur effect in Windows 10 if the acrylic effect is not available.

You can composite blur and gradient effect with background color too
```
background
{
    color=#fff
    opacity=20
    effect=2 // blur effect 
    gradient
    {
        enabled = 1
        linear = [0, 100, 0, 0]
        stop = [
            [0, #00f, 10],
            [.5, #0f0, 40],
            [1, #f00, 50]
        ]
    }
}
```
![gradient ](https://user-images.githubusercontent.com/39198/221662143-2462d47b-c5dc-4f20-9ee9-76476aac3e8a.png)

**Reply by 0-BlackSpectrum-0** [reply]
> Acrylic
> 
> ```
> background
> {
>     opacity=0
>     effect=[3, color.accent, 50]
> }
> ```
> 
> Save changes, press `CTRL` + `RIGHT-CLICK` or restart Explorer to reload settings.
> 
> https://nilesoft.org/docs/syntax/set

 Can you tell me how do I change the color as well? color.accent is used to get the color so how do I change it?

**Reply by moudey** [reply]
>Can you tell me how do I change the color as well? color.accent is used to get the color so how do I change it?
```
background
{
    opacity=0
    effect=[3, #00ff00, 50]  // [3, tint color, opacity value from 0 to 100]
}
```

**Reply by 0-BlackSpectrum-0** [reply]
**Thanks**

**Comment by JZersche** [comment]
Possible to use a background image with a path for the context menu background, or only gradient/colors are supported currently?

**Reply by RubicBG** [reply]
only gradient/colors
